### ISSUE TITLE
This thing is genuinely great.

### ISSUE THREAD

**Issue by urbanawakening** [github-for-unity/Unity#1106]
While the title does conflict with the nature of issues, I can safely say that what follows will not.
This thing is fuckin' great, no, not because it manages to execute its four basic primary functions, but because how easily it can just break for no reason at all. You wanna pull from origin? Nope, simply not possible because you decided to make multiple branches so you and your team wouldn't have to commit to master all the fucking time and make a mess of the git history. You wanna fix that one extremely annoying "refusing to merge unrelated histories" error, nah, the command line feature doesn't even fucking work either so you can't allow merging with unrelated histories. It'd be comical how this thing manages to break 75% of the time if not extremely rage inducing.
In short, thank you developers, you've created something truly horrible. It's not everyday that you find something that makes you truly wonder "Who the fuck thought this was okay?"
